_posts/2017-03-03-ome-users-meeting-2017.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Speakers announced for OME Users Meeting 2017
+intro-blurb: The OME team is pleased to announce the draft programme for the OME Users Meeting 2017, to be held in Dundee on 31st May to June 2nd.
+---
+The draft programme for the OME Users Meeting 2017 is now available at 
+({{ site.baseurl }}/community/events/12th-annual-users-meeting-2017.html). 
```

Comment:
You need some link text inside square brackets to format the link properly e.g.

`The draft programme for the [OME Users Meeting 2017]({{ site.baseurl }}/community/events/12th-annual-users-meeting-2017.html) is now available.`